### ISSUE TITLE
Mention updating Flatpaks in Quick Setup

### ISSUE THREAD

**Issue by Qronikarz** [flatpak/flatpak.github.io#590]
Would it be worth it to mention how to update Flatpaks in a chosen distro? For example Linux Mint only recently in the 21.1 version added Flatpak updates to update manager. You needed to either run `flatpak update` or enable the automatic option in the settings before so it's possible there are people that don't know that they need to update their Flatpaks. Of course that wouldn't help if someone never reads about it on the site, but when someone stumbles onto the setup site and sees the `Flatpak support is built into, no setup required!` one might assume updates are also included.

Also a question: are the Flatpak updates itself worth mentioning? Newest on Mint is currently 1.12.7 and I don't know if there are any advantages to try to install newer versions for normal, daily usage.

**Comment by sourcejedi** [comment]
I think if there's distro-specific update instructions, they really should be included.  It's a pity because (security) updates are why I like flatpak over AppImage.

It could also use improving for Ubuntu, IMO.

For Ubuntu Desktop (and debian on x86 default desktop), what should happen is that in step 2, installing `gnome-software-plugin-flatpak`, will provide update notifications through gnome-software.  And gnome-software autostarts on login.

The difficulty for Ubuntu is, the quick setup steps don't highlight this.  They only say step 2 "makes it possible to install apps without needing the command line".  This is bad, if you just install a specific flatpak as a one-off and don't know to think about security updates.  You might even have deliberately skipped step 2 on Ubuntu, because you read it installs gnome-software as a second software app alongside Ubuntu Software.

**Comment by AsciiWolf** [comment]
Do the update notifications from deb gnome-software work on Ubuntu?

**Comment by sourcejedi** [comment]
Ah.  Bad choice of words.  It's a family PC, I don't use it and can't say when I last saw a notification.  I've just re-checked, and I'm pretty sure the updates are getting *installed*.

Gnome Software can install updates manually.  I've removed Ubuntu Software.  I don't know for certain everything works as well, if you leave Ubuntu Software installed.

I did the same on an older Ubuntu as well, probably 20.04... I make a point of checking updates and I think that worked well.  (The machine was a pain, but that's a different story).

Edit: Update notifications were in my head because of a previous problem, but that doesn't have anything to do with flatpak.

**Comment by AsciiWolf** [comment]
I think that the information about "flatpak update" could be added into the Manual Install details on Flathub.org. You can try creating a ticket about it [here](https://github.com/flathub/website/issues).

**Comment by sourcejedi** [comment]
For me, adding `flatpak update` underneath the list of commands in "Manual Install" details would not be helpful.

I don't want to know how to update manually, and I would just look at flatpak help for that.

The question is how to be sure my system provides automatic updates and/or update notifications for security e.g. of Firefox.  For Ubuntu, I'm happy the linked setup instructions did what I wanted.  However the instructions don't actually say that they will do what I want, and the step that does what I want says there is a small disadvantage and sounds like an optional step.

This is more specific and different to the situation described on Mint, so if you don't think there's enough common ground then of course I could split out an new issue.

**Comment by AsciiWolf** [comment]
I agree, however on the other hand adding workarounds and additional information into the quick setup steps just because some distribution has broken Flatpak support is also not ideal, at least in my opinion.
